readthedocs/redirects/models.py

```diff
@@ -174,8 +171,8 @@ def get_redirect_path(self, path, language=None, version_slug=None):
 
     def redirect_prefix(self, path, language=None, version_slug=None):
         if path.startswith(self.from_url):
-            log.debug('Redirecting...', redirect=self)
-            cut_path = re.sub('^%s' % self.from_url, '', path)
+            log.debug("Redirecting...", redirect=self)
+            cut_path = path[len(self.from_url) :]
```

Comment:
Looks like our linter doesn't agree with black on this one, the linter doesn't want a space before the `:`.